datafusion/sqllogictest/test_files/window.slt

```diff
@@ -3493,6 +3493,48 @@ select sum(1) over() x, sum(1) over () y
 ----
 1 1
 
+query TT
```

Comment:
Could you please leave a comment explaining what this is a test for (specifically that you are expecting that the nth value should be negative)? 

Also I can't see a negative nth value in the plan so maybe I am missing something

Reply:
Sure added a comment. However, since window expression names are string, without string manipulation it is not easy to reflect window reversal in the plan. However, I think we should do this absolutely. I will fix this problem in subsequent PRs.